### DISCUSSION TITLE
Baked shadow plane should be ignored when generating USDZ

### DISCUSSION THREAD

**Discussion by colordrop** [google/model-viewer#2434] [Ideas]
When generating usdz of models with baked floor shadows, the floor plane should be ignored. Now the plane looks too dark and with good quality contact shadow on iOS could be invisible.
![B3909397-E71A-4BBB-9E4C-9A5CC233C741](https://user-images.githubusercontent.com/16795504/119990932-3432ac80-bfc9-11eb-9648-e184a2c9c934.jpg)
![85717121-8DB8-49DC-A3FD-00F7168C3CF4](https://user-images.githubusercontent.com/16795504/119990936-34cb4300-bfc9-11eb-9f0f-5c7d7c5ba4c8.png)

**Comment by elalish** [comment]
Yes, this is a limitation of the auto-conversion, so you may need to supply your own USDZ, or remove the shadow and settle for our auto-generated one. This might be avoidable once our scene-graph API matures a bit. @timmmeh
